src/Nova/Location.php

```diff
@@ -45,6 +55,9 @@ public function fields(Request $request)
             nova('market') ? BelongsTo::make('Market', 'market', nova('market'))->required() : null,
             Text::make('Name')->required(),
             Slug::make('Slug')->from('Name'),
+            TextCopy::make('Link',  function () {
+                return config('app.url') . config('tipoff.web.uri_prefix') . $this->path;
```

Comment:
We should probably create another accessor that returns the full webpage url for the path.